rust/src/core.rs

```diff
@@ -38,6 +39,36 @@ pub const STREAM_TOCLIENT: u8 = 0x08;
 pub const STREAM_GAP:      u8 = 0x10;
 pub const STREAM_DEPTH:    u8 = 0x20;
 pub const STREAM_MIDSTREAM:u8 = 0x40;
+pub const DIR_TOSERVER:    u8 = 0b0000_0100;
```

Comment:
Why did you not use STREAM_TOSERVER ?

Reply:
I thought that keeping it might not enforce people to use Direction enum in the latest PRs. If you don't agree, I can rename these back to STREAM_TOSERVER

Reply:
Smart :-)